tests/Feature/MeetingTest.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Meeting;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MeetingTest extends TestCase
+{
+    use RefreshDatabase, WithFaker;
+
+    public function test_create()
+    {
+        $organizer = factory(User::class)->create();
+
+        $startTime = $this->faker->dateTimeBetween('now', '+2 years');
+        $endTime = $this->faker->dateTimeBetween(
+            $startTime->format('Y-m-d H:i:s').' +1 hours',
+            $startTime->format('Y-m-d H:i:s').' +6 hours'
+        );
+
+        $response = $this->actingAs($organizer, 'api')->postJson(
+            route('api.meeting.create'),
+            [
+                'name' => $this->faker->company,
+                'description' => $this->faker->paragraph,
+                'location' => $this->faker->address,
+                'start_time' => $startTime->format('Y-m-d H:i:s'),
+                'end_time' => $endTime->format('Y-m-d H:i:s')
+            ]
+        );
+
+        $response
+            ->assertStatus(201)
+            ->assertJson([
+                'success' => true
+            ]);
```

Comment:
Can we also assert that the meeting gets created? We should be able to do something like `$messing = Meeting::first()` then `$this->assertEquals('Foo Name', $meeting->name)` etc.